src/App.js

```diff
@@ -1,232 +1,69 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import Carousel from "./components/Carousel";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import ProductDetails from "./components/ProductDetails";
+import ProductInfo from "./components/ProductInfo";
+import PropTypes from "prop-types";
+import SideBar from "./components/SideBar";
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      imageWasClicked: false,
+      newImage: props.list.products[1].imgUrl,
```

Comment:
Little detail: You might potentially get an empty array as props. Then this will fail. 
Best practice: When accessing indexes of arrays, always make sure they exist.